### DISCUSSION TITLE
Where have the latest version of the install kit gone

### DISCUSSION THREAD

**Discussion by ReidarMidtun** [codeproject/CodeProject.AI-Server#140] [Q&A] [answered]
I have developed an app that uses the CodeProject.AI for image analizes. My install script downloads the latest CodeProject.AI install kit from https://www.codeproject.com/ai/latest.aspx. After the shutdown of CodeProject, this does not work anymore. Is there a new url for the lastest installer?

**Comment by MikeLud** [comment, marked as answer]
Try this url
https://codeproject.github.io/

**Reply by ReidarMidtun** [reply]
Thanks for pointing me to https://codeproject.github.io/!

But, I need a way to find the latest windows installer. From the page I see this is currently "https://www.codeproject.com/KB/Articles/5322557/CodeProject.AI_Server-win-x64-2.8.0.zip". Since the version is coded into the file name, it will change. By using the old system(https://www.codeproject.com/ai/latest.aspx), the file name of the latest version was handled dynamic. So my question is, will there be a similar mechanism implemented an available from https://codeproject.github.io/.

**Reply by Sailing74** [reply]
You can use https://api.github.com/repos/codeproject/CodeProject.AI-Server/releases/latest to find the latest version, and use that version to compile the file name for scripts, etc. (curl -s https://api.github.com/repos/codeproject/CodeProject.AI-Server/releases/latest | jq -r .tag_name)
